src/AppInstallerRepositoryCore/Microsoft/Schema/1_4/Interface.h

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+#pragma once
+#include "Microsoft/Schema/ISQLiteIndex.h"
+#include "Microsoft/Schema/1_3/Interface.h"
+#include "Microsoft/Schema/1_2/Interface.h"
+
+namespace AppInstaller::Repository::Microsoft::Schema::V1_4
+{
+    // Interface to this schema version exposed through ISQLiteIndex.
+    struct Interface : public V1_3::Interface
```

Comment:
>Interface [](http://example.com/codeflow?start=11&length=9)

You also need to implement `CheckConsistency` here and call into the same for the dependencies table.